bite-size-notes/week-8-a-Probability Review-Zhijie Tan

```diff
@@ -0,0 +1,204 @@
+
+This section covers the introduction of symbols related to Probability and discusses Random variables in different formulas. 
+
+##
+The following notes will explain the meanings of symbols and demonstrate different formulas related to random variables.
+
+- Probability Distribution: An introduction to probability distributions, which describe how probabilities are distributed over the values of a random variable.
+
+- Example of Probability Mass Function (PMF): A discussion and example of PMFs, which apply to discrete random variables and give the probability that a discrete random variable is exactly equal to some value.
+
+- Example of Probability Density Function (PDF): A discussion and example of PDFs, which apply to continuous random variables and describe the likelihood of a random variable to take on a particular value.
+
+- Rules Corresponding to Random Variables: An overview of some fundamental rules and properties related to random variables.
+
+### Random Variable
+
+### Symbol
+- X:random varible(RV). a quantity whose value is unknown/uncertain 
+- x: possible value of X or sample of X or realization of X 
+
+
+### Probability Space ($\Omega$,$F$,$\mathcal{P}$)
+- $\Omega$: sample space (all possible outcomes of RV)
+- $\mathcal{F}$: Event space (all possible sets of outcomes in $\Omega$)
+- $\mathcal{P}$: Probability measure (assigns a probability to each other)
+
+## Moving forward consider X v P(X) where P(x) is a probabilty distribution
+- Expectation $\mathbb{E}$
+  
+  ${\mathbb{E} \text{xp} [g(x)]} = \int_x g(x) P(x) \, dx \text{ or } \sum_x g(x) P(x)$
+  - mean $\mathcal{u_x}$ = ${\mathbb{E} \text{xp}(x) [g(x)]}$
+
+
+
+### Discrete Random Variable Plot
+
+The following plot represents the probability mass function (PMF) of a discrete random variable: 
+![Discrete Random Variable Plot](PMF.png)
+- $S=[1.2.3]$
+- P($x\in$,$\mathcal{S}$)= $\sum_{x=S}P(x)$
+- $\sum_xP(x)=1$ 
+- $0≤P（x）≤1$
+
+### Continous Random Variable Plot
+The following plot represents the probability density function (PDF) of a continuous random variable:
+
+![Discrete Random Variable Plot](PDF.png)
+
+- P($x\in$,$\mathcal{S}$)= $\int_\mathcal{S}P(x) dx$
+- $\int_x P(X)=1$
+  
+### The probability density function of a multivariate normal (or Gaussian) distribution
+$N(u, \Sigma) = \frac{1}{(2\pi)^{k/2} \det(\Sigma)^{1/2}} \exp\left(-\frac{1}{2}(x - u)^T \Sigma^{-1} (x - u)\right)$
+
+- $u$: the mean vector
+- $\Sigma$:the covariance matrix
+- $k$:the dimensionality of the distribution
+- $x$:the vector of the variables
+- $det(\Sigma)$:the determinant of the the covariance matrix 
+- $\Sigma^{-1}$is the inverse of the covariance matrix
```

Comment:
Good point, I have clearly recognized the meaning of the probability density function here.

Reply:
Thank you!